ruby/rails/security/brakeman/check-permit-attributes-high.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+- id: check-permit-attributes-high
+  patterns:
+  - pattern: $P.permit($ATTRIBUTE)
+  - metavariable-regex:
+      metavariable: $ATTRIBUTE
+      regex: .*(admin|account_id).*
+  message: Calling `permit` on security-critical properties like `$ATTRIBUTE` may
+    leave your application vulnerable to mass assignment.
```

Comment:
Would you mind adding a line about what alternatives there are to permit?

Reply:
`permit` is an "allow the user to alter this field" operation.  The guidance from Brakeman is simply "do not do this, it is bad"

Reply:
ah I see 🤔